docs/reference/search/request/highlighting.asciidoc

```diff
@@ -1,9 +1,7 @@
 [[search-request-highlighting]]
 === Highlighting
 
-Allows to highlight search results on one or more fields. The
-implementation uses either the lucene `plain` highlighter, the
-fast vector highlighter (`fvh`) or `postings` highlighter.
+Allows to highlight search results on one or more fields.
```

Comment:
"Highlighters allow you to produce highlighted snippets from one or more fields in your search results."